scripts/squad-run-node.sh

```diff
@@ -111,31 +118,35 @@ init_dir_lite() {
 
 init_env_db-lookup() {
     echo "===== init env db-lookup ====="
-    echo "CVM_ID=$CVM_ID"
+    echo "LH_ID=$LH_ID"
     echo "CBS_ID_0=$CBS_ID_0"
     echo "CBS_ID_1=$CBS_ID_1"
     echo "CBS_ID_2=$CBS_ID_2"
     echo "CBS_ID_FLOAT=$CBS_ID_FLOAT"
     # install python p7zip
     # yum install -y https://repo.ius.io/ius-release-el$(rpm -E '%{rhel}').rpm
-    yum install -y python3 p7zip.x86_64 screen
+    yum install -y -q python3 p7zip.x86_64 screen
 
     # install tccli
-    pip3 install tccli
+    pip3 install -q tccli
 }
 
 # input:
```

Comment:
I don't see the init-disk.sh script launch, where is it done?

Reply:
I just use the function `attach_and_mount` which contains the steps of disk format, mount and /etc/fstab append.
It seems ok. What do you think of it?

Reply:
if parted works ok, no need to use init-disk anymore, simply remove it from there

Reply:
I think we can keep it, be useful some day.